frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -2122,44 +2158,82 @@ https://archiveweb.page/images/${"logo.svg"}`}
-                    ? humanizeSchedule(utcSchedule)
-                    : msg("Invalid date")}</span
-                >.`,
-            )}
-          </p>
-          <p>
+      ${when(
+        this.formState.scheduleFrequency,
+        () =>
+          html`${inputCol(html`
+            <btrix-time-input
+              hour=${ifDefined(this.formState.scheduleTime?.hour)}
+              minute=${ifDefined(this.formState.scheduleTime?.minute)}
+              period=${ifDefined(this.formState.scheduleTime?.period)}
+              @time-change=${(e: TimeInputChangeEvent) => {
+                this.updateFormState({
+                  scheduleTime: e.detail,
+                });
+              }}
+            >
+              <span slot="label">${msg("Start Time")}</span>
+            </btrix-time-input>
+            ${scheduledDate(this.utcSchedule)}
+          `)}
+          ${this.renderHelpTextCol(
+            msg(`A crawl will run at this time in your current timezone.`),
+          )}`,
+        () => html`
+          ${inputCol(html`
+            <sl-input
+              name="scheduleCustom"
+              label=${msg("Cron Schedule")}
+              placeholder="@hourly"
+              value=${ifDefined(this.formState.scheduleCustom)}
+              minlength="6"
+              @sl-change=${(e: SlChangeEvent) => {
+                const input = e.target as SlInput;
+                const value = (e.target as SlInput).value;
+                const parsed = parseCron(value);
+
+                if (!value) return;
+
+                if (parsed) {
+                  input.helpText = "";
+                  input.setCustomValidity("");
+                } else {
+                  let errorMessage = "";
+
+                  try {
+                    cronstrue.toString(value);
+                  } catch (err) {
+                    if (typeof err === "string") {
+                      // TODO Handle localized error?
+                      errorMessage = err.replace("Error: ", "");
+                    }
+                  }
+
+                  errorMessage =
+                    errorMessage ||
+                    msg("Please fix invalid Cron expression syntax.");
+
+                  input.helpText = errorMessage;
+                  input.setCustomValidity(errorMessage);
+                }
+              }}
+              required
+            >
+            </sl-input>
+            ${scheduledDate(this.formState.scheduleCustom)}
+          `)}
+          ${this.renderHelpTextCol(html`
+            ${msg("Specify a schedule in Cron format.")}
             ${msg(
-              html`Next scheduled run:
-                <span
-                  >${utcSchedule
-                    ? humanizeNextDate(utcSchedule)
-                    : msg("Invalid date")}</span
-                >.`,
+              html`Supports Unix cron syntax and certain macros like
+                <code>@hourly</code> and <code>@yearly</code>.`,
```

Comment:
Might be worth noting in this `msg`'s description that `@hourly` and `@yearly` shouldn't be translated